README.md

```diff
@@ -1,14 +1,28 @@
 # OpenTelemetry Lambda
+![GitHub Java Workflow Status](https://img.shields.io/github/workflow/status/open-telemetry/opentelemetry-lambda/Continuous%20Build%20(Java)/main?label=CI%20%28Java%29&style=for-the-badge)
+![GitHub Collector Workflow Status](https://img.shields.io/github/workflow/status/open-telemetry/opentelemetry-lambda/Continuous%20Build%20(Collector)/main?label=CI%20%28Collector%29&style=for-the-badge)
+![GitHub NodeJS Workflow Status](https://img.shields.io/github/workflow/status/open-telemetry/opentelemetry-lambda/Continuous%20Build%20(NodeJS)/main?label=CI%20%28NodeJS%29&style=for-the-badge)
+![GitHub Terraform Lint Workflow Status](https://img.shields.io/github/workflow/status/open-telemetry/opentelemetry-lambda/Continuous%20Build%20(Terraform)/main?label=CI%20%28Terraform%20Lint%29&style=for-the-badge)
+![GitHub Python Rull Request Workflow Status](https://img.shields.io/github/workflow/status/open-telemetry/opentelemetry-lambda/Pull%20Request%20(Python)/main?label=Pull%20Request%20%28Python%29&style=for-the-badge)
 
+## [OpenTelemetry Collector extension](collector/README.md)
```

Comment:
I think this naming is a bit more clear? If we are generic we can give an overview that calls out all the layers :)

```suggestion
## OpenTelemetry Lambda Layers
```

Reply:
Done.